src/feature/consulenze/components/ReservationModal.tsx

```diff
@@ -87,6 +78,13 @@ export const ReservationModal: React.FC<RouteComponentProps> = () => {
       } catch (error) {
         gotoStep("error");
       }
+      if (userAppointment && !isEmpty(userAppointment)) {
+        try {
+          await deleteAppointment(userAppointment[0].id);
+        } catch (error) {
+          console.log(error);
+        }
+      }
```

Comment:
Conviene tu lo metta `try catch` all'interno del `try` in cui prenoti l'appuntamento.

Reply:
@leonardodemarco00 fixed questo ?

Reply:
T